aws/warehouse.go

```diff
@@ -80,6 +88,96 @@ type AccountWarehouse struct {
 	svcUserArn *string
 	store      storage.Store
 	tmp        map[string]iam.AccessKey
+	keyGC      *processaws.KeyGC
+}
+
+func (wh *AccountWarehouse) GetServiceAccounts(ctx context.Context, project string) (<-chan *clouds.Account, error) {
+	c := make(chan *clouds.Account)
+	go func() {
+		defer close(c)
+		f := func(acct *iam.User) error {
+			a := &clouds.Account{
+				ID:          aws.StringValue(acct.UserId),
+				DisplayName: aws.StringValue(acct.UserName),
+			}
+			select {
+			case c <- a:
+			case <-ctx.Done():
+				return ctx.Err()
+			}
+			return nil
+		}
+		sess, err := createSession()
+		if err !=nil {
+			return
+		}
+		svc := iam.New(sess)
+		// FIXME: get PathPrefix from config
+		accounts, err := svc.ListUsers(&iam.ListUsersInput{
+			PathPrefix: aws.String("/ddap/"),
+		})
+		users := accounts.Users
+		for _, user := range users {
+			if err := f(user); err != nil {
+				glog.Errorf("getting user accounts list: %v", err)
+				return
+			}
+		}
+
+	}()
+	return c, nil
+}
+
+func (wh *AccountWarehouse) RemoveServiceAccount(ctx context.Context, project, accountID string) error {
```

Comment:
Can we get rid of this method while the functionality is unimplemented, or is it required for some interface?

Reply:
It is required for an interface